posthog/temporal/data_imports/pipelines/schemas.py

```diff
@@ -53,6 +53,7 @@
     ExternalDataSource.Type.GOOGLEADS: (),
     ExternalDataSource.Type.TEMPORALIO: TEMPORALIO_ENDPOINTS,
     ExternalDataSource.Type.DOIT: (),
+    ExternalDataSource.Type.MONGO: (),
```

Comment:
Nit: Can the type name `MONGO` be the same as the value `MongoDB` - so `ExternalDataSource.Type.MONGODB`?